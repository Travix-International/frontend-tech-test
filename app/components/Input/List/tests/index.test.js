import React from 'react';
import { create } from 'react-test-renderer';

import InputList from '../index';

const initial = {};
const renderComponent = props => {
  const finalProps = { ...initial, ...props };

  return (
    <InputList {...finalProps} />
  );
};

describe('<InputList />', () => {
  it('should render component', () => {
    const tree = create(renderComponent()).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should render component when disabled', () => {
    const props = {
      disabled: true
    };
    const tree = create(renderComponent(props)).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
