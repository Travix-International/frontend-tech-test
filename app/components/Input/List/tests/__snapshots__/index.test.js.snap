// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputList /> should render component 1`] = `
<input
  className="List__Input-kOyobO iWGCK"
  onChange={[Function]}
/>
`;

exports[`<InputList /> should render component when disabled 1`] = `
<input
  className="List__Input-kOyobO dDgqIv"
  disabled={true}
  onChange={[Function]}
/>
`;
